libusb/os/windows_hotplug.c

```diff
@@ -0,0 +1,296 @@
+#include "libusbi.h"
+#include "threads_windows.h"
+
+#include "windows_common.h"
+#include "windows_hotplug.h"
+
+#include <stdio.h>
+#include <dbt.h>
+#include <usbiodef.h>
+
```

Comment:
It could help future contributors to have a short description of how this module works. Here is what I came up with.

// The Windows Hotplug system is a three steps process.
// 1. We create a monitor on GUID_DEVINTERFACE_USB_DEVICE via a hidden window.
// 2. Upon notification of an event, we run the current windows backend to get the list of devices. This updates the hotplug status of each device to one of three values {connected, newly_connected, disconnected}. 
// 3. According to the value, we generate events to libusb client via hotplug callbacks.

Reply:
Thanks!